docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 ---
-version: '2'
+version: '3'
```

Comment:
Why?? Docker compose file format is at version 2. Why do you increase the version here?